platforms/jvm/language-java/src/integTest/groovy/org/gradle/api/tasks/JavaExecWithLongCommandLineIntegrationTest.groovy

```diff
@@ -61,14 +61,14 @@ class JavaExecWithLongCommandLineIntegrationTest extends AbstractIntegrationSpec
 
             tasks.register("runWithExecOperations") {
                 dependsOn sourceSets.main.runtimeClasspath
-                def runExecutable = run.executable ? run.executable : null
+                def runExecutable = run.executable
```

Comment:
💅 Can we inline this now?

Reply:
Unfortunately not, we get:
```
Execution failed for task ':runWithExecOperations'.
> Could not get unknown property 'run' for object of type org.gradle.process.internal.DefaultJavaExecAction_Decorated.
```
with configuration cache, but it works in vintage mode.